src/util/expandThoughts.js

```diff
@@ -80,31 +82,36 @@ export const expandThoughts = (path, thoughtIndex, contextIndex, contextViews =
   return (isOnlyChildNoUrl || isTable() || pinChildren() || publishPinChildren
     ? children
     : children.filter(child => {
+      const childPath = (path || []).concat(child)
```

Comment:
Have you tested this in the context view? I'm skeptical that this works in all cases, but I could be wrong. The `child` is not necessarily the last item in a path. It seems like we might be mixing up `Path` types with `Context` types here.